master/internal/rm/kubernetesrm/spec.go

```diff
@@ -361,6 +369,19 @@ func (p *pod) configurePodSpec(
 	if podSpec.ObjectMeta.Labels == nil {
 		podSpec.ObjectMeta.Labels = make(map[string]string)
 	}
+	// add determined metadata as pod labels
+	podSpec.ObjectMeta.Labels[userLabel] = p.submissionInfo.taskSpec.Owner.Username
+	podSpec.ObjectMeta.Labels[workspaceLabel] = p.submissionInfo.taskSpec.Workspace
```

Comment:
checkpointGC tasks aren't associated to a workspace. Should we not set the label or leave it empty?

Reply:
As far as I can tell, it's valid to have an empty string as the label value. If a task is not assigned to a workspace, I think it makes sense to leave it empty; that's still informative.